### ISSUE TITLE
Any document if want to make similar tool from start in nodejs

### ISSUE THREAD

**Issue by RYG81** [jagenjo/webglstudio.js#60]
Is there any document available for making similar tool in nodejs?

**Comment by jagenjo** [comment]
nodejs is not a language, you mean javascript? my recomendation is that you start by creating a 3D engine, there are many books about coding 3D engines.

**Comment by RYG81** [comment]
Yes JavaScript but Node.js as backend in place of PHP.  Also what library will be better use ThreeJS, Babylon or your libraries.

**Comment by jagenjo** [comment]
oh, you want to replace litefilesystem then, not webglstudio, yeah, it shouldnt be complex, in the litefilesystem repo you have more info.
